cpp/log/etcd_consistent_store_test.cc

```diff
@@ -289,8 +289,8 @@ TEST_F(EtcdConsistentStoreTest, TestSetServingSTHWontOverwriteWithOlder) {
   EXPECT_EQ(util::error::OUT_OF_RANGE, status.CanonicalCode()) << status;
 }
 
-
-TEST_F(EtcdConsistentStoreDeathTest, TestSetServingSTHChecksInconsistentSize) {
+TEST_F(EtcdConsistentStoreDeathTest,
+       TestSetServingSTHChecksInconsistentSizeDeathTest) {
```

Comment:
From what I can tell here, it runs all the tests in `FooDeathTest`, *then* all the tests in `FooTest`. For example:

```
[==========] Running 5 tests from 2 test cases.
[----------] Global test environment set-up.
[----------] 1 test from ThreadPoolDeathTest
[ RUN      ] ThreadPoolDeathTest.AddingMoreTasksAfterClosedGoesBang

[WARNING] /usr/src/gmock/gtest/src/gtest-death-test.cc:825:: Death tests use fork(), which is unsafe particularly in a threaded context. For this test, Google Test couldn't detect the number of threads.
[       OK ] ThreadPoolDeathTest.AddingMoreTasksAfterClosedGoesBang (200 ms)
[----------] 1 test from ThreadPoolDeathTest (200 ms total)

[----------] 4 tests from ThreadPoolTest
[ RUN      ] ThreadPoolTest.Delay
[       OK ] ThreadPoolTest.Delay (200 ms)
[ RUN      ] ThreadPoolTest.DelayDoesNotBlockAThread
[       OK ] ThreadPoolTest.DelayDoesNotBlockAThread (200 ms)
[ RUN      ] ThreadPoolTest.NaturalOrderingPreserved
[       OK ] ThreadPoolTest.NaturalOrderingPreserved (201 ms)
[ RUN      ] ThreadPoolTest.CancelsDelayTasks
[       OK ] ThreadPoolTest.CancelsDelayTasks (0 ms)
[----------] 4 tests from ThreadPoolTest (601 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 2 test cases ran. (802 ms total)
[  PASSED  ] 5 tests.
```

The documentation seems to imply that there's something forcing you to put `EXPECT_DEATH` checks only in test cases named `*DeathTest`, but I just tried it, and it works (there's not even a warning).

So it's possible that we might have a stray `EXPECT_DEATH` in a test that doesn't conform to the naming, and that could very well throw things out of whack...